model/areStatus.js

```diff
@@ -188,6 +188,10 @@
 	returnObj.getStatus = function() {
 		return status;
 	}
+
+     returnObj.isConnected = function() {
+         return status && status != ACS.statusType.CONNECTING && status != ACS.statusType.CONNECTIONLOST && status != ACS.statusType.DISCONNECTED;
```

Comment:
yes `if(status)` is the same as `if(status != null && status != undefined)`. In every boolean expression it should be the first check to avoid any errors. So if you would e.g. check for a field like `status.name.length` the expression should look like `if(status && status.name && status.name.length > 1)`.